### PR TITLE
PR: modification of article router for add middleware in backend

### PR DESCRIPTION
**Pull request by creative2113**
Hello, dear admin and gentlemen.
I have updated middleware handle in article controller.
please check.
best regards.

### CODE REVIEW

**Review by creative2113 — COMMENTED** [review]
looks good

**Review by creative2115 — APPROVED** [review]
good job

File: server/src/routes/articleRoutes.js
[the remark this answers is not in the document]
Comment (creative2113) [reply server/src/routes/articleRoutes.js:1]: thank you

**Review by creative2115 — APPROVED** [review]
that is very good.
go ahead.

**Review by shinevue — APPROVED** [review]
You are king of coding.

**Review by silvershiny — APPROVED** [review]
Ok

**Review by goldsunshines — APPROVED** [review]
Dear creative
Welcome you

**Review by shiny7star — APPROVED** [review]
Funny

**Review by techietrend — APPROVED** [review]
Awesome

**Review by creative2115 — APPROVED** [review]
very good.
go ahead.

**Comment by BlackGhost2693** [comment]
good
you are great

**Pull request merged**